newbank/server/NewBankClientHandler.java

```diff
@@ -204,10 +217,16 @@ private void printMenu2(CustomerID customer){
 		out.println("1. Check your accounts");
 		if(!bank.hasAccount(customer, "Main")){
 			out.println("2. Open new Main account");}
+		if(bank.hasAccount(customer, "Main")){
+			out.println("2. View Main account");}
```

Comment:
These could be done using `else` instead - to avoid a bit of code duplication.

For instance:

```
if(bank.hasAccount(customer, "Main")){
    out.println("2. View Main account");
} else {
    out.println("2. Open new Main account");
}
```

Or, you could extract that into a method, something like `printMenu2Option(customer, account)`, to de-duplicate even further.